README.md

```diff
@@ -65,3 +65,31 @@ npm run build
 ```sh
 npm start
 ```
+
+## DB Connection
+
+1. Modify the <.env> file with the new DB information. These will be provided. Ensure the <.env> file is included in the <.gitignore> file.
+
+2. Run this command to connect to the database.
+
+```sh
+psql -h dpg-cqjdupaj1k6c739r0mrg-a.virginia-postgres.render.com -U cyf_tms_user -d cyf_tms_cjao
+```
+
+3. Run this command to execute the SQL Script after connecting to the database. Replace the </path/to> with the setup_production.sql path
+
+```sh
+\i /path/to/setup_production.sql
+```
+
+4. Run this command to create a DB backup. Replace the </path/to/backup/> with the path where the backup should be created
+
+```sh
+pg_dump -h dpg-cqjdupaj1k6c739r0mrg-a.virginia-postgres.render.com -U cyf_tms_user -d cyf_tms_cjao -F c -b -v -f /path/to/backup/cyf_tms_cjao_backup.backup
+```
+
+5. Run this command to restore the backup file. Replace the </path/to/backup/> with the backup file path
+
+```sh
+pg_restore -h dpg-cqjdupaj1k6c739r0mrg-a.virginia-postgres.render.com -U cyf_tms_user -d cyf_tms_cjao -v /path/to/backup/cyf_tms_cjao_backup.backup
+```
```

Comment:
We don't need any of this, we just need to make sure the migrations command is run before we start the server in render. We also need to recreate the DB in render (because our current credentials have been leaked) and make sure the correct environment variables are set in production